content/methods/future-methods/cell-key-perturbation.jsonnet

```diff
@@ -0,0 +1,18 @@
+{
+  title: 'Cell Key Perturbation',
+  date: '2024-02-26',
+  contact_details: 'smlhelp@ons.gov.uk',
+  method_metadata: {
+    'Expert group': 'Statistical Disclosure Control',
+    Theme: 'Statistical Disclosure Control',
+    Author: 'ONS Community Method',
+    'Languages': 'Python/R',
```

Comment:
As per the ticket, languages must be "Python & R" this has been agreed with methodology. The reason we have Python/PySpark is because the main language is Python but it is using the pyspark library. Cell Key Perturbation is written in Python and R.